packages/openneuro-app/src/scripts/datalad/dashboard/datasets/dataset-tab.jsx

```diff
@@ -71,7 +72,7 @@ const DatasetTab = ({
       </div>
       <div className="filters-sort-wrap clearfix">
```

Comment:
The "Filter by:" label overlaps with the sort-by input element when the page width is 690-700 px.